docs/field_tests.rst

```diff
@@ -121,7 +122,21 @@ Test Scenarios
     **Workflow:** 
 
     * In tags.conf for each tag defined in the stanza, the plugin generates a test case.
-    * For each tag, the plugin generates a search query including the stanza and the tag and asserts event_count > 0
+    * For each tag, the plugin generates a search query including the stanza and the tag and asserts event_count > 0.
+
+**7. Search query should be present in each savedsearches.**
+
+    .. code-block:: python
+
+        test_savedsearches[<savedsearch_stanza>]
+
+    Test case verifies that there are savedsearches mapped with the search. 
```

Comment:
Please update the description. we are not checking the mapping, we are validating that the search produces a result which is mentioned in the saved search

Reply:
Done.